src/cloud/project/privatelink-service.md

```diff
@@ -64,7 +64,7 @@ Enabling PrivateLink can take up to 5 business days. Providing incomplete, or in
 
 ### Enablement workflow
 
-The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}.
+The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}.  Before you submit your support ticket, please fill out the following excel file with the necessary information. [PrivateLink Information Template] (Excel file you can download, has sample data)
```

Comment:
```suggestion
The following workflow outlines the enablement process for PrivateLink integration with {{site.data.var.ece}}. Before you submit your support ticket, please fill out the following Excel file with the necessary information. [PrivateLink Information Template] (The Excel file, which you can download, has sample data)
```

Is it correct to do this as an XLS sheet? Two issues - 1. Not everyone uses Excel or can read it 2. People might be concerned about it carrying a virus - what's the stance on these two?

Reply:
Good questions @barny. We do not store these type of binary files in our repo. We recently removed several large binaries that were taking up space in our repo and added them to a `downloads/` directory on the site. 

Two things come to mind @jpanke:

- Hosting this file somewhere other than devdocs.magento.com where the internal owners can manage changes would be ideal. That way, we can simply link to it from devdocs and any future changes will be available on devdocs as soon as owners update the linked doc.
- If hosting elsewhere is not possible, then the file should be converted to PDF that contains a "last modified date" somewhere in the document itself (not the filename) and uploaded to the `downloads/` directory by someone on the `devdocs-admins` team. The source file must be maintained by the owner on internal systems.

Reply:
I think you meant that comment for @BarnyShergold

Reply:
I did. Sorry about that.

Reply:
S'okay - I still get the email 😄